api/bibliotheca.py

```diff
@@ -195,23 +197,32 @@ def fulfill(self, patron, password, pool, internal_delivery):
         )
         if drm_scheme == DeliveryMechanism.FINDAWAY_DRM:
             fulfill_method = self.get_audio_fulfillment_file
-            # The provided media type is application/json, which
-            # is too vague for us.
-            force_content_type = DeliveryMechanism.FINDAWAY_DRM
+            content_transformation = self.findaway_license_to_webpub_manifest
```

Comment:
I see, so this is where you start the webpub manifest off with the information from the opds feed?

Reply:
Not the OPDS feed per se -- the same underlying information from the database that is also used to populate the OPDS feed.